client/mapView/MapRendererContext.cpp

```diff
@@ -266,6 +272,16 @@ bool MapRendererAdventureContext::showBlocked() const
 	return settingShowBlocked;
 }
 
+bool MapRendererAdventureContext::showSpellRange(const int3 & position) const
+{
+	if (!settingSpellRange)
+		return false;
+
+	auto hero = adventureInt->curHero();

```

Comment:
Check that both adventureInt and hero are not nullptr?

Reply:
adventureInt - always exists
curHero - *should* always exist, will add check just in case